src/network/clientpackethandler.cpp

```diff
@@ -48,6 +48,22 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "particles.h"
 #include <memory>
 
+const char *accessDeniedStrings[SERVER_ACCESSDENIED_MAX] = {
+	N_("Invalid password"),
+	N_("Your client sent something the server didn't expect.  Try reconnecting or updating your client."),
+	N_("The server is running in simple singleplayer mode.  You cannot connect."),
```

Comment:
unrelated but I like this better
```suggestion
	N_("The server is running in singleplayer mode.  You cannot connect."),
```